src/main/java/ru/hogwarts/school/controller/AvatarController.java

```diff
@@ -0,0 +1,64 @@
+package ru.hogwarts.school.controller;
+
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import ru.hogwarts.school.model.Avatar;
+import ru.hogwarts.school.service.AvatarService;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+@RestController
+@RequestMapping("avatar")
+public class AvatarController {
+    private final AvatarService avatarService;
+
+    public AvatarController(AvatarService avatarService) {
+        this.avatarService = avatarService;
+    }
+
+    @PostMapping(value = "/{id}/avatar", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public ResponseEntity<Avatar> upLoadAvatar(@PathVariable Long id, @RequestParam MultipartFile avatar) throws IOException {
+        avatarService.uploadAvatar(id, avatar);
+        return ResponseEntity.ok().build();
+    }
```

Comment:
```suggestion
    public ResponseEntity<Long> upLoadAvatar(@PathVariable Long id, @RequestParam MultipartFile avatar) throws IOException {
        Long avatarId = avatarService.uploadAvatar(id, avatar);
        return ResponseEntity.ok(avatarId);
    }
```
Сделай так, чтобы тело было типа Long